run_generate.py

```diff
@@ -164,9 +164,12 @@ def main():
         tokenizer.padding_side = "left"
         tokenizer.truncation_side = "left"  # TODO @nouamane: do we want this?
         dummy_inputs = [
-            # "Passage: Daniel went back to the garden. Mary travelled to the kitchen. Sandra journeyed to the kitchen. Sandra went to the hallway. John went to the bedroom. Mary went back to the garden. Where is Mary?\nAnswer:",
+            "The future of AI is",
+            "Passage: Daniel went back to the garden. Mary travelled to the kitchen. Sandra journeyed to the kitchen. Sandra went to the hallway. John went to the bedroom. Mary went back to the garden. Where is Mary?\nAnswer:",
             "def fib(n)",
-            # "This film was probably inspired by Godzilla",
+            "This film was probably inspired by Godzilla",
+            "Advancements in technology will lead to",
+            "Tomorrow's world is shaped by",
```

Comment:
maybe add a very long prompt to test as well?
You can maybe get one from here: https://huggingface.co/datasets/HuggingFaceTB/cosmopedia/viewer/web_samples_v2/train?f[text_token_length][min]=2274&f[text_token_length][imax]=2504